plugins/contrast/index.js

```diff
@@ -39,9 +39,23 @@ class ContrastPlugin extends Plugin {
             suggestedColorsRatio: suggestedColors.contrast
         };
 
+        // Add click handler to preview checkbox.
+        let $description = $(descriptionTemplate(templateData));
+        $description.find(".previewCb").click(function() {
+            if(this.checked) {
```

Comment:
nit: `if (`

We should probably add `space-after-keywords: [2, "always"]` to `.eslintrc`